lib/mayaUsd/python/wrapPrimWriter.cpp

```diff
@@ -320,21 +319,130 @@ const UsdMayaJobExportArgs& unprotect_GetExportArgs(UsdMayaPrimWriter& pw)
 {
     return reinterpret_cast<PrimWriterAllowProtected&>(pw)._GetExportArgs();
 }
-boost::python::object unprotect_GetSparseValueWriter(UsdMayaPrimWriter& pw)
+MayaUsdLibSparseValueWriter unprotect_GetSparseValueWriter(UsdMayaPrimWriter& pw)
 {
-    return boost::python::object(
+    return MayaUsdLibSparseValueWriter(
```

Comment:
Wrapping the sparse value writer because that class can not be passed-by-value, which is the default in Python. See the wrapper include file for details.